lib/tools/apk-utils.js

```diff
@@ -524,6 +524,9 @@ apkUtilsMethods.install = async function install (appPath, options = {}) {
       output : `${output.substr(0, 150)}...${output.substr(output.length - 150)}`;
     log.debug(`Install command stdout: ${truncatedOutput}`);
     if (/\[INSTALL[A-Z_]+FAILED[A-Z_]+\]/.test(output)) {
+      if (this.isTestPackageOnlyError(output)) {
+        log.warn('Set `allowTestPackages` capability to true in order to allow test packages installation.');
+      }
       throw new Error(output);
```

Comment:
please add the hint to the actual error description here

Reply:
@mykola-mokhnach you mean a link to the capability? or our documentation?

Reply:
I mean 

```
if (this.isTestPackageOnlyError(output)) {
        const msg = `Set 'allowTestPackages' capability to true in order to allow test packages installation.`;
        log.warn(msg);
        throw new Error(`${output}\n${msg}`);
      }
```